.travis.yml

```diff
@@ -24,3 +24,6 @@ script:
   - mv config/npc/config.specific.sample.php config/npc/config.specific.php
   - composer run start:integration-services
   - composer run test
+
+after_success:
+    - bash <(curl -s https://codecov.io/bash)
```

Comment:
What is this doing?

Reply:
It uploads the output `test/coverage.xml` file to the smrealms/smr project on codecov.io. I'll add a comment saying so.

Reply:
You're not specifying that file anywhere though o_O

Reply:
Wait, it's downloading a script or something and then executing it in bash? How does it know where to grab the xml file?

Reply:
It's magic. I think it scans for valid coverage files. I was thinking about explicitly adding in the `-f tests/coverage.xml` option so it's a little more clear/explicit about what it's doing (and probably will do that now), but it's not necessary.

Reply:
Oh ok

Reply:
Yeah, it's downloading a bash script.